omas/omas_plot.py

```diff
@@ -1186,7 +1186,7 @@ def core_transport_fluxes(ods, time_index=None, fig=None, show_total_density=Tru
 
     def sum_density_types(specie_index):
         final_density = numpy.zeros(len(prof1d['grid.rho_tor_norm']))
-        for therm_fast in ['', '_thermal', '_fast']:
+        for therm_fast in ['_thermal', '_fast']:
```

Comment:
@TimSlendebroek I'd run `ods.physics_core_profiles_densities()` to make sure that `density`, `density_thermal`, and `density_fast` for electrons and ions are filled and are self-consistent.